draft-ietf-oauth-v2-1.md

```diff
@@ -233,13 +233,13 @@ the application.  This creates several problems and limitations:
    the end-user's password and all of the data protected by that
    password.
 
-With OAuth, an end user (resource owner) can grant a printing
-service (client) access to their protected photos stored at a photo-
-sharing service (resource server), without sharing their username and
-password with the printing service.  Instead, they authenticate
-directly with a server trusted by the photo-sharing service
-(authorization server), which issues the printing service delegation-
-specific credentials (access token).
+With OAuth, an end user (resource owner) can grant a financial management 
+service (client) access to their sensitive transaction history stored at 
+a banking service (resource server), without sharing their username and 
```

Comment:
I'd suggest using _credentials_ instead of `username and password` in this sentence.

The reason is credentials also cover things such as API keys.

Reply:
This is an example, and so we are specific with username and password. 

There are many, many kinds of credentials. Access tokens and refresh tokens are credentials. A broader term muddies understanding the eample.

Reply:
Got it.